validator/src/main.rs

```diff
@@ -1,3 +1,3 @@
 fn main() {
     solana_validator::main()
-}
```

Comment:
Oh where did the change to validator/src/ go that sets `enable_ctrl_c_handler = true`